kanalony-storage-logic/src/main/scala/kanalony/storage/logic/Queries.scala

```diff
@@ -9,9 +9,11 @@ object Queries {
   object HourlyUserActivityPrtnEntryClstCountry extends HourlyUserActivityPrtnEntryClstCountryQuery
   object HourlyUserActivityPrtnEntryQuery extends HourlyUserActivityPrtnEntryQuery
   object HourlyUserActivityClstEntryQuery extends HourlyUserActivityClstEntryQuery
+  object DailyUserActivityPrtnEntryQuery extends DailyUserActivityPrtnEntryQuery
   val queries : List[IQuery] = List(
```

Comment:
Since we don't need to know the concrete queries, how about if each query will register itself to the list of available queries?

// in DailyUserActivityPrtnEntryQuery.scala
object DailyUserActivityPrtnEntryQuery extends DailyUserActivityPrtnEntryQuery {
  kanalony.storage.logic.Queries.add(this)
}